core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -396,7 +396,7 @@ export class CreateServiceComponent implements OnInit {
       this.typeOfPlatform = platformType;
       if(this.typeOfPlatform == 'azure'){
         if(this.typeOfService == 'sls-app'){
-          this.changeServiceType('api')
+          this.changeServiceType('function')
```

Comment:
What's the purpose of this code? It is confusing since you are setting the service type as `function` when it is `sls-app`?

Reply:
This was done on purpose based on edge case. If user selects sls-app service type and then selects AWS, it's fine. But if he wishes to change the platform from AWS to AZURE, the serviceType is still sls-app, which is not supported, that's why under this case, the serviceType is default to API, but since we are disabling api so by default it is function.

Reply:
This needs to be cleaned up. If we do just top down filtering of options rather than going both ways.